docs/package.json

```diff
@@ -41,9 +41,10 @@
     "eslint-plugin-prettier": "^2.2.0",
     "highlight.js": "^9.12.0",
     "if-env": "^1.0.0",
+    "json-loader": "^0.5.7",
```

Comment:
Actually, I think that we don't need this:

> Since webpack >= v2.0.0, importing of JSON files will work by default. You might still want to use this if you use a custom file extension. See the v1.0.0 -> v2.0.0 Migration Guide for more information

Reply:
yeah read this later...